README.il.md

```diff
@@ -0,0 +1,802 @@
+# Alpine.js
+
+![npm bundle size](https://img.shields.io/bundlephobia/minzip/alpinejs)
+![npm version](https://img.shields.io/npm/v/alpinejs)
+[![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg?sanitize=true)](https://alpinejs.codewithhugo.com/chat/)
+
+Alpine.js ti offre la natura reattiva e dichiarativa di grandi framework come Vue o React a un costo molto inferiore.
+
+Puoi mantenere il tuo DOM e cospargere il comportamento come ritieni opportuno.
```

Comment:
Literal translations are not a good idea. This sentence (cospargere il comportamento) doesn't make sense in Italian.
Same for other sentences below. It sounds like it's been automatically translated, would you mind reviewing and adjust any sentence that doesn't make much sense? Thanks